import-core/import-services/src/main/java/org/devgateway/importtool/model/User.java

```diff
@@ -13,6 +13,8 @@
 @JsonAutoDetect (fieldVisibility = JsonAutoDetect.Visibility.ANY, isGetterVisibility = JsonAutoDetect.Visibility.ANY, getterVisibility = JsonAutoDetect.Visibility.ANY)
 @Entity
 @Table (name = "user_account")
+// FIXME This is not in use remove before closing the feature
```

Comment:
Please create a follow up ticket to check all FIXME/TODO in the code that could be resolved after merging.